recipe/meta.yaml

```
@@ -10,25 +10,29 @@ source:
   sha256: 412b92d3a28211dc5ea8191fc1d95832759e1f24548bc2e8d7c4c0d1d92b904c
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # pymc not available for py38
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.8
-    - setuptools >=61.0
+    - python
+    - setuptools >=61.0.0
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - arviz >=0.13.0
     - matplotlib-base >=3.5.1
-    - numpy >=1.17
+    - numpy {{ numpy }}
     - pandas
-    - pymc >=5.3.0
+    - pymc >=5.5.0
```

```suggestion
    - pymc >=5.3.0
```
Should be >=5.3.0 and comment also says that the minimum version needed https://github.com/pymc-labs/pymc-marketing/blob/ee587a8992d1bcffb616cb9638b340e80ca2edb6/pyproject.toml#L20